mscore/palette.h

```diff
@@ -51,6 +55,8 @@ struct PaletteCell {
       double yoffset { 0.0   };      // in spatium units of "gscore"
       qreal mag      { 1.0   };
       bool readOnly  { false };
+      Palette* parent;
```

Comment:
What's the purpose of "parent" here ? 
If possible, I would prefer to set the parent when creating the cell, and not having a getParent with a loop, and a method in the struct.